### PR TITLE
Updating the descriptions, titles, and README

### PR DESCRIPTION
**Pull request by benjmountain**
Updating a handful of descriptions and titles

### CODE REVIEW

File: step.yml [hunk 1]
```diff
@@ -13,14 +13,20 @@ summary: |
   Upload APK/AAB/IPA artifacts to Sauce Labs.
 description: |
   Upload your build files to Sauce Labs for remote device testing.
-  Needs an APK, AAB, or IPA file, along with Sauce Labs username and access key.
+  Needs an APK, AAB, or IPA file along with Sauce Labs username and access key.
```
Comment (camseda) [inline step.yml:16]: and app name if we're making it required here

File: step.yml [hunk 2]
```diff
@@ -69,9 +75,10 @@ inputs:
   - sauce_app_name: $SAUCE_APP_NAME
     opts:
       title: Sauce Labs App Name
-      summary: Your Sauce Labs App Name
+      summary: Your Sauce Labs application name and extension.
       description: |
         This is translated to the File Name in Sauce Labs.
         Can be used to activate a specific app with: `storage:filename=<fileName>.<ext>`
```
Comment (camseda) [inline step.yml:81]: Confused by storage: here -- and can you add an example? like 'AndroidApp.apk'

File: step.yml [hunk 3]
```diff
@@ -69,9 +75,10 @@ inputs:
...
-        rather than using the File ID automatically generated for an app.
-        If this is not present, File Name will be defaulted by the app's name in `$UPLOAD_PATH`
\ No newline at end of file
+        rather than using the File ID automatically generated on upload.
+        If this is not present, File Name will be defaulted by the app's name in `$UPLOAD_PATH`
```
Comment (camseda) [inline step.yml:83]: Maybe put this first like. 'Defaults to the app's name in $UPLOAD_PATH'. Can be used to activate etc etc.

**Review by camseda — CHANGES_REQUESTED** [review]

File: step.yml (hunk 2 above)
Comment (benjmountain) [reply step.yml:81]: This is a reference to using `filename` in capabilities from their docs - https://docs.saucelabs.com/mobile-apps/app-storage/#file-name-instead-of-file-id. Happy to add doc link, specific example, or just remove

File: step.yml (hunk 3 above)
Comment (benjmountain) [reply step.yml:83]: I'll remove this for now since we're requiring it and we can add it back when this isn't a required field

**Pull request merged**